### PR TITLE
Fix checkbox sizes

### PR DESCRIPTION
**Pull request by SerhiiTsybulskyi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Screenshot 2024-03-26 at 15 16 22

Issue Number: N/A

## What is the new behavior?
Screenshot 2024-03-26 at 15 15 29

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by amcdnl** [comment]
@SerhiiTsybulskyi - Should we change the font size too?

**Comment by SerhiiTsybulskyi** [comment]
> @SerhiiTsybulskyi - Should we change the font size too?

Yes, I'm updating PR to change label font size

**Comment by SerhiiTsybulskyi** [comment]
Screenshot 2024-03-26 at 15 24 54

File: src/form/Checkbox/CheckboxTheme.ts [hunk 1]
```diff
@@ -5,6 +5,11 @@ export interface CheckboxTheme {
   label: {
     base: string;
     clickable: string;
+    sizes: {
```
Comment (amcdnl) [inline src/form/Checkbox/CheckboxTheme.ts:8]: Need to update docs

Comment (SerhiiTsybulskyi) [reply src/form/Checkbox/CheckboxTheme.ts:8]: It's already updated.
That using ThemeRender which using variable
`<ThemeRender theme={darkCheckboxTheme} />`

Screenshot 2024-03-26 at 15 38 54

**Pull request merged**